SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/SpatialViewUtils.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "ComponentTestUtils.h"
+#include "SpatialView/OpList/EntityComponentOpList.h"
+#include "SpatialView/ViewDelta.h"
+#include "SpatialView/WorkerView.h"
+
+namespace SpatialGDK
+{
+Worker_EntityId TestEntityId = 1;
```

Comment:
These should all be const.
As they are global I'd also give them slightly longer names. E.g. SpatialViewTestEntityId.

However if any of these can be defined in an non namespace in the relevant cpp file that would be much better.

Reply:
Looking at them I think they can all be const file scope rather than global.